manifests/assisted-image-service/assisted-image-service-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  namespace: assisted-installer
+  name: assisted-image-service
+  annotations:
+    cert-manager.io/issuer: assisted-installer-ca
+    nginx.ingress.kubernetes.io/backend-protocol: "HTTPS"
```

Comment:
So if we don't have this because we're not using nginx ingress what needs to happen?
Can we not use https in our services then?

Reply:
You are right @jianzzha, I forgot that. The assisted image service needs that in order to talk to successfully connect to the assisted service API. I will update the patch tomorrow.

Reply:
I think it should be something like this:

```diff
diff --git a/manifests/assisted-image-service/assisted-image-service-statefulset.yaml b/manifests/assisted-image-service/assisted-image-service-statefulset.yaml
index 0d42e0d..5587c0a 100644
--- a/manifests/assisted-image-service/assisted-image-service-statefulset.yaml
+++ b/manifests/assisted-image-service/assisted-image-service-statefulset.yaml
@@ -30,8 +30,8 @@ spec:
           value: /etc/image-service/certs/tls.crt
         - name: HTTPS_KEY_FILE
           value: /etc/image-service/certs/tls.key
-        - name: HTTPS_CA_FILE
-          value: /etc/image-service/ca-bundle/service-ca.crt
+        - name: ASSISTED_SERVICE_API_TRUSTED_CA_FILE
+          value: /etc/image-service/ca-bundle/ca.crt
         - name: ASSISTED_SERVICE_SCHEME
           value: https
         - name: ASSISTED_SERVICE_HOST
@@ -105,9 +105,9 @@ spec:
         secret:
           defaultMode: 420
           secretName: assisted-image-service
-      - configMap:
+      - secret:
           defaultMode: 420
-          name: assisted-image-service
+          secretName: assisted-service
         name: service-cabundle
       - name: image-service-data
         emptyDir: {}
```

I replaced also the `HTTPS_CA_FILE` environment variable with `ASSISTED_SERVICE_API_TRUSTED_CA_FILE` because the name is more informative, and the former is deprecated:

https://github.com/openshift/assisted-image-service/blob/88a5bd5bdef0eaf32095d1b9af0fd0083f6047a6/main.go#L31-L32

I will test this tomorrow and update the pull request.

Reply:
> You are right @jianzzha, I forgot that. The assisted image service needs that in order to successfully connect to the assisted service API. I will update the patch tomorrow.

Ah I see. That make sense.